DigitalOcean.API/Clients/IImagesClient.cs

```diff
@@ -10,6 +10,13 @@ public interface IImagesClient {
         /// </summary>
         Task<IReadOnlyList<Image>> GetAll(ImageType type = ImageType.All);
 
+        /// <summary>
+        /// To create a new custom image.
+        /// The image must be in the raw, qcow2, vhdx, vdi, or vmdk format.
+        /// It may be compressed using gzip or bzip2 and must be smaller than 100 GB after being decompressed.
+        /// </summary>
+        Task<Image> Create(Models.Requests.Image image);
```

Comment:
Since only the Update method (below) is currently implemented it made the most sense to rename that method's class to `UpdateImage` and use `Models.Requests.Image` for this new Create method.